pkg/dockerregistry/server/repositorymiddleware.go

```diff
@@ -178,7 +178,7 @@ func newRepositoryWithClient(
 	if err != nil {
 		context.GetLogger(ctx).Error(err)
 	}
-	pullthrough, err := getBoolOption(PullthroughEnvVar, "pullthrough", false, options)
+	pullthrough, err := getBoolOption(PullthroughEnvVar, "pullthrough", true, options)
```

Comment:
does this break backward compatibility of this setting? iow. pullthrough is now enabled by default.